### PR TITLE
:sparkles: handle /

### PR DESCRIPTION
**Pull request by pyaillet**
Like proposed in #1

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -9,6 +9,28 @@ import (
...
+<p>
+Give a username and get an avatar in return: <a href="https://avatars.charlyx.dev/twitter?username=charlyx">https://avatars.charlyx.dev/twitter?username=charlyx</a>
+</p>
+
+<p>
+You can ask for variant sizings such as "bigger", "mini" and "original" (default size being "normal").
+</p>
+
+<p>
+Example : <a href="https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger">https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger</a>
```
Comment (charlyx) [inline server/server.go]:
```suggestion
Example: <a href="https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger">https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger</a>
```

File: server/server.go [hunk 2]
```diff
@@ -9,6 +9,28 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 )
 
+const usage = `<html><head><title>Not found</title></head><body>
```
Comment (charlyx) [inline server/server.go]: It'd be nice to display an emoji as favicon using this technique: https://css-tricks.com/emojis-as-favicons/

Maybe something like:
```html
<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>👤</text></svg>">
```

**Review by charlyx — CHANGES_REQUESTED** [review]

File: server/server.go [hunk 3]
```diff
@@ -9,6 +9,28 @@ import (
...
+
+<p>
+Example: <a href="https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger">https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger</a>
+</p>
+</body>
+</html>`
+
+func defaultHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusNotFound)
+	fmt.Fprintf(w, usage)
```
Comment (charlyx) [inline server/server.go]: `http.Error(w, usage, http.StatusNotFound)` might do the exact same thing using one line.

Comment (pyaillet) [reply server/server.go]: `http.Error` wraps the content with a full html page and `<pre></pre>` markup
I'll go with `w.Write()`

File: server/server.go [hunk 4]
```diff
@@ -9,6 +9,28 @@ import (
...
+
+<p>
+Example: <a href="https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger">https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger</a>
+</p>
+</body>
+</html>`
+
+func defaultHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusNotFound)
+	w.Write([]byte(usage))
```
Comment (charlyx) [inline server/server.go]:
```suggestion
	fmt.Fprint(w, usage)
```

**Pull request merged**